projects/laji/src/app/shared-modules/select/metadata-select/metadata-select.component.ts

```diff
@@ -165,6 +167,22 @@ export class MetadataSelectComponent implements OnChanges, OnDestroy, ControlVal
     this.cd.markForCheck();
   }
 
+  deleteOptionsfromExcludedOptions(options, excludeOptions) {
+    if (!excludeOptions.length) {
+      return options;
+    }
+
+    options = options.reduce((filtered, current) => {
+      const id = current['id'].replace('MX.', '');
+      if (excludeOptions.indexOf(id) === -1) {
+        filtered.push(current);
+      }
+      return filtered;
+    }, []);
+
+    return options;
```

Comment:
The replace 'MX' refers to taxon object and in the context of metadata select there shouldn't be anything about it.  Ideally you could just use simple 
```typescript
return options.filter(options => !excludeOptions.includes(option.id))`
```

Reply:
@vixriihi ExcludedOptions are without MX. prefix, so options will be the same after filtering. I moved the change to upper component species-form, but i don't think my solution is good.

Reply:
there seems to be at least 3 places now where the MX is added to the value.  In the `species-form.compoent.ts` it might be better to just rename all invasiveStatuses as they really are (since that real key is already visible in the query parameter)